okta/table_okta_group_rule.go

```diff
@@ -0,0 +1,141 @@
+package okta
+
+import (
+	"context"
+	"strings"
+
+	"github.com/okta/okta-sdk-golang/v2/okta"
+	"github.com/okta/okta-sdk-golang/v2/okta/query"
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableOktaGroupRule() *plugin.Table {
+	return &plugin.Table{
+		Name:        "okta_group_rule",
+		Description: "Retrieve group rules for Okta. Group rules define conditions and actions for automating group membership.",
+		Get: &plugin.GetConfig{
+			Hydrate:           getOktaGroupRule,
+			KeyColumns:        plugin.SingleColumn("id"),
+			ShouldIgnoreError: isNotFoundError([]string{"Not found"}),
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listOktaGroupRules,
+		},
+		Columns: []*plugin.Column{
```

Comment:
Could you please include the common columns for this table, as we do for other tables? For reference, see: [link](https://github.com/turbot/steampipe-plugin-okta/blob/main/okta/table_okta_app_assigned_group.go#L36).